prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -803,6 +803,29 @@ public async Task<ActionResult<Site>> DeclineSite(int siteId)
             return Ok(ApiResponse.Result(updatedSite));
         }
 
+        // PUT: api/Sites/5/enable-editing
+        /// <summary>
+        /// Enable editing a site
+        /// </summary>
+        /// <param name="siteId"></param>
+        [HttpPut("{siteId}/enable-editing", Name = nameof(EnableEditingSite))]
+        [Authorize(Roles = Roles.EditSite)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        public async Task<ActionResult<Site>> EnableEditingSite(int siteId)
```

Comment:
I've been pushing to slowly stop using the typed versions of `ActionResult` unless we actually use it. I'd prefer if this was `public async Task<ActionResult> EnableEditingSite(int siteId)`

Reply:
@johnnyl6 PR looks good if we can get this addressed we can get it merged in

Reply:
done in da81eee